### PR TITLE
Add a documentation related to the volume's filesystem permission errors

### PR DESCRIPTION
**Pull request by cortexx**
Add a temporary diagnostic documentation to allow our customers to be able to fix the permission errors when enabling persistence data in volume.

This documentation can be used until a patch is released on our production environments (ETA early 2022).

### CODE REVIEW

File: pages/platform/kubernetes-k8s/fix-persistent-volumes-permissions/guide.en-asia.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+Be able to autonomously fix an OVHcloud Managed Kubernetes Service when `Permission Errors` are encountered during Helm Chart deployment or deployment creation.  
+
+## Issue explanation
+
+Several Helm charts are updated with some security hardening best practices.  
+Using a `non-root` container for example is a new rule to follow for security reason.  
+But a main drawback to use non-root containers is related to mounting persistent volumes in these containers.  
+Indeed, processes running inside these containers do not have the necessary privileges to modify the ownership of the existing filesystem in a volume.  
+
+A solution is to use the `SecurityContext` provides by Kubernetes to automatically modify the ownership of the attached volumes and to provide a `StorageClass` which support to modify the volume's filesystem.
```
Comment (XavierDuthil) [inline pages/platform/kubernetes-k8s/fix-persistent-volumes-permissions/guide.en-asia.md]:
```suggestion
A solution is to use the `SecurityContext` provided by Kubernetes to automatically modify the ownership of the attached volumes and to provide a `StorageClass` which supports modifying the volume's filesystem.  
```

File: pages/platform/kubernetes-k8s/fix-persistent-volumes-permissions/guide.en-asia.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+2) You are using the Bitnami Helm Charts and you want to be able to quickly fix this behavior without waiting our patch.  
+You could follow the instructions described in this documentation: https://docs.bitnami.com/kubernetes/faq/troubleshooting/troubleshooting-helm-chart-issues/
+
+3) /!\ This solution is not recommended if you don't know what you are doing and only works with clusters above `1.20` version /!\
+You are impacted by this issue but your Helm Chart provider didn't offer a proper solution and you can't wait our official patch.  
+
+If you are in this case, please follow these instructions at your own risk:
+
+- Verify what is the `StorageClass` that you are using by default (generaly the `csi-cinder-high-speed`):
```
Comment (XavierDuthil) [inline pages/platform/kubernetes-k8s/fix-persistent-volumes-permissions/guide.en-asia.md]:
```suggestion
- Verify what is the `StorageClass` that you are using by default (generally the `csi-cinder-high-speed`):
```

File: pages/platform/kubernetes-k8s/fix-persistent-volumes-permissions/guide.en-asia.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+- Delete the concerned Helm Chart
+
+For example with the Helm Chart `bitnami/wordpress` which is concerned by this behavior:
+
+```bash
+➱ helm uninstall my-first-k8s-wordpress
+```
+
+And don't forget to verify if concerned `PersistentVolumeClaim` and `PersistentVolume` have been deleted before to reinstall the Helm Chart:
```
Comment (XavierDuthil) [inline pages/platform/kubernetes-k8s/fix-persistent-volumes-permissions/guide.en-asia.md]:
```suggestion
And don't forget to verify if concerned `PersistentVolumeClaim` and `PersistentVolume` have been deleted before reinstalling the Helm Chart:
```

File: pages/platform/kubernetes-k8s/fix-persistent-volumes-permissions/guide.en-asia.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+
+- Reinstall the concerned Helm Chart or deployment
+
+For example with the Helm Chart `bitnami/wordpress` which is concerned by this behavior:
+
+```bash
+➱ helm install my-first-k8s-wordpress bitnami/wordpress
+```
+
+You can see, that the pods are now up and running, which means that the permission errors related to the `persistantVolumes` is now fixed.
```
Comment (XavierDuthil) [inline pages/platform/kubernetes-k8s/fix-persistent-volumes-permissions/guide.en-asia.md]:
```suggestion
You can see, that the pods are now up and running, which means that the permission errors related to the `persistentVolumes` is now fixed.
```

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thanks for your contribution @cortexx & @XavierDuthil

**Pull request merged**